atomic_reactor/schemas/user_params.json

```diff
@@ -61,6 +67,7 @@
     "release": {"type": "string"},
     "scratch": {"type": "boolean"},
     "signing_intent": {"type": "string"},
+    "tags_from_yaml": {"type": "boolean"},
```

Comment:
Is this in addition to them being in container_yaml?

Reply:
Yes for now.  I don't have a strong opinion on whether its better to break them out or not.